src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (message: string) => void;
+};
+
+export const ErrorMessage: React.FC<Props> = React.memo(
+  ({
+    errorMessage,
+    setErrorMessage,
+  }) => {
+    return (
+      <div className={
+        classNames(
+          'notification is-danger is-light has-text-weight-normal',
+          {
+            hidden: !errorMessage,
```

Comment:
hidden is not hiding element instantly. it fades away, and we cant remove error text while it fades away. But its just my opinion)